src/Record.purs

```diff
@@ -177,6 +226,13 @@ merge l r = runFn2 unsafeUnionFn l r
 -- |
 -- | ```purescript
 -- | union { x: 1, y: "y" } { y: 2, z: true }
+-- | -- { x: 1, y: "y", z: true }
+-- | ```
+-- |
+-- | More generally:
+-- |
+-- | ```purescript
+-- | union { x: 1, y: "y" } { y: 2, z: true }
```

Comment:
Given that you can't actually have a record value with duplicate fields (though having a _type_ with duplicate fields is fine), should there be an extra warning here about that?

Reply:
Probably. How should the warning be worded? And should it appear after the first paragraph's sentence? If so, the docs would read:

> Merges two records with the first record's labels taking precedence in the case of overlaps. Unlike `merge`, this does not remove duplicate labels from the resulting record type. This can result in better inference for some pipelines, deferring the need for a `Nub` constraint. **WARNING SENTENCE GOES HERE**